partitioner/src/main/java/io/confluent/connect/storage/partitioner/PartitionerConfig.java

```diff
@@ -61,8 +65,8 @@
       "This configuration is used to set the format of the data directories when partitioning with "
       + "``TimeBasedPartitioner``. The format set in this configuration converts the Unix timestamp "
       + "to proper directories strings. For example, if you set "
-      + "``path.format='year'=YYYY/'month'=MM/'day'=dd/'hour'=HH/``, the data directories will have"
-      + " the format ``/year=2015/month=12/day=07/hour=15``.";
+      + "``path.format='year'=YYYY/'month'=MM/'day'=dd/'hour'=HH``, the data directories will have"
```

Comment:
I wasn't feeling very strong about this being added to the doc. However it's there to denote that it's a directory path. 

Re: this property I added specific checks to make sure that `//` is avoided in S3.